Resources/ConfigPresets/L5/L5.toml

```diff
@@ -35,7 +38,7 @@ space_wind   = true
 #grid_impulse = true # this just sends some salvage wrecks flying.
 
 [shuttle]
-arrivals                       = false # doesn't work very well :c
+arrivals                       = false # doesn't work very well in general, and not at all with Outpost
```

Comment:
We can use a specific Docking Airlock prototype that comes with the correct docking tag. This is a quickfix an admeme can do once there is somewhere for the arrivals shuttle to dock on the new station.

That said, the Outpost gamemode probably works best without the arrivals shuttle anyways. While the "doesn't work very well" comment is very outdated and is no longer true, leaving it false by default (until an admeme places a proper arrivals dock and changes the cvar in-round) makes the most sense.

Reply:
"Doesn't work well" refers to the shuttle FTLing sideways, in circles, or stationary in space. If you would like the comment removed, though, we can do that.

Reply:
Honestly, I completely forgor about that bug. Nevermind.